src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="searchForm_" >
```

Comment:
```suggestion
    <div class="searchForm" >
```
Why do you need this underscores?